.golangci.yml

```diff
@@ -3,7 +3,7 @@ new: true
 new-from-rev: 722af08
 run:
   concurrency: 6
-  deadline: 5m
+  timeout: 5m
```

Comment:
This should be different PR?

Reply:
Yeah I've split it off https://github.com/openshift/kube-compare/pull/38